scripts/convert-embedding-results.py

```
@@ -0,0 +1,271 @@
+    if args.model:
+        print(f"Filtering by model: {args.model}")
+
+    rows = scan_results_directory(args.results_dir, args.model)
+
+    if rows:
+        write_csv(rows, args.output)
+
+        # Print summary
+        models = set(r['model'] for r in rows)
+        test_types = set(r['test_type'] for r in rows)
+        print(f"\nSummary:")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)